index.js

```diff
@@ -133,23 +134,24 @@ const processSpecialCommands = ( queryType, domSlice, processedArg ) => {
 	let text;
 	switch ( queryType ) {
 		case '!condition':
-			const segments = domSlice.outerHTML.split( '<h2 class="title">' ); // split html by condition title tags
+			const segments = domSlice.outerHTML.split( '<b>' ); // split html by condition title tags
```

Comment:
The "Confused" status on the target page has a nested instance of a `<b/>` tag to title a table of confusion rolls.
You could change the split string to `"<br /> <b>"` to split at "titles" only, but that still leaves that condition with a table that needs formatting later.

Ex: https://discord.com/channels/738456503266312262/759856360053276684/850513517593362472

Reply:
I pushed a super hacky fix to get the table...